src/popup/popup.js

```diff
@@ -31,6 +47,10 @@ const handleInternalMessage = (message) => {
     return;
   }
 
+  if (message.type === 'active-identity-response-error') {
+    handleError('Unable to retrieve IDP from WebID.');
```

Comment:
1. Mention the profile so users know what profile the extension is talking about.
2. The error message should disappear when a working profile is selected.